### DISCUSSION TITLE
ATIS downlink INOP? (atis confirmed up and running)

### DISCUSSION THREAD

**Discussion by OddSobriquet** [flybywiresim/aircraft#2322] [Help]
I'm experiencing some troubles with the atis downlink (vatsim).
It gives me a D-ATIS not available, ATIS confirmed up and working. confirmed correct source.

![image](https://user-images.githubusercontent.com/72888738/101681840-0e3d3e00-3a63-11eb-83ea-7d62359c04a2.png)
![image](https://user-images.githubusercontent.com/72888738/101678151-d7185e00-3a5d-11eb-94d7-dab97f2931ed.png)
(timestamp not equal. FMS result did not change when vpilot screencap was added)
![image](https://user-images.githubusercontent.com/72888738/101678171-e13a5c80-3a5d-11eb-8f73-c6eccc6869cb.png)

manifest for version attached
[manifest.txt](https://github.com/flybywiresim/a32nx/files/5668209/manifest.txt)

**Comment by ImenesFBW** [comment]
The station you request ATIS from needs to be opened on VATISM, or there is no station to pull from.

**Reply by OddSobriquet** [reply]
as I mentioned above: ATIS confirmed up and working. confirmed correct source.
It has worked in the past days, can't say which built that was, though.

edit: some seem to work. not-working :LOWW, EGKK.

**Reply by ImenesFBW** [reply]
> as I mentioned above: ATIS confirmed up and working. confirmed correct source.
> It has worked in the past days, can't say which built that was, though.
> 
> edit: some seem to work. not-working :LOWW, EGKK.

When I was answering you didn't include the first pic. Are you sure the ATIS station was up when you requested? Works for me when the station is online. Are you sure the ATIS was up when you first sent it?

![image](https://user-images.githubusercontent.com/71195324/101684350-92dd8b80-3a66-11eb-8c9b-9eceb7703816.png)

**Reply by OddSobriquet** [reply]
yes.
I had the same in LOWW.
I did double check if it was working, since it didn't I didn't bother updating the fms screen cap

**Comment by flightsimsimon** [comment]
Means there is no atis online but you can still request a metar report

**Reply by OddSobriquet** [reply]
as I mentioned above: Atis is up and running.
Not all seem to be affected. Not working EGKK, LOWW.

**Comment by Annolago** [comment]
I have also had issues with getting ATIS reports for EDDF, EDDH, ESMS, EKCH, EKAH, EKBI.
I have tried switching between all sources but with no luck. So if anyone have an idea on how to fix it would be apricated especially when we are about to get the printer function..
